pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -438,8 +438,9 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsWindow* dialog, QWidget*
 		dialog->registerWidgetHelp(m_ui.PCRTCOverscan, tr("Show Overscan"), tr("Unchecked"),
 			tr("Enables the option to show the overscan area on games which draw more than the safe area of the screen."));
 
-		dialog->registerWidgetHelp(
-			m_ui.fmvAspectRatio, tr("FMV Aspect Ratio"), tr("Off (Default)"), tr("Overrides the full-motion video (FMV) aspect ratio."));
+		dialog->registerWidgetHelp(m_ui.fmvAspectRatio, tr("FMV Aspect Ratio"), tr("Off (Default)"),
+			tr("Overrides the full-motion video (FMV) aspect ratio.<br>"
+			   "If disabled aka default, it will inherent same value from above Aspect Ratio value which if also default is 'Auto Standard (4:3/3:2 Progressive)' or other value if changed there as well."));
```

Comment:
Way too wordy; took me a couple reads to understand what you're trying to say.
Cut the fat. Something like "If disabled, the FMV Aspect Ratio will be the same as the general Aspect Ratio" gets the point across.
The user already knows the defaults; the UI tells them.

Maybe even changing the setting to "FMV Aspect Ratio Override" makes more sense.

Reply:
I'll change it to be better for users in terms of readability.

Reply:
Okay changed it.